2-mandatory/4-people-i-know.js

```diff
@@ -402,16 +418,29 @@ This time, I only want the full names of the people who can multitask
 */
 
 let colleaguesWhoCanMultitask = [];
+let coll = [];
+for (let i in friends) {
```

Comment:
plz use the array functions' instead of the for loop in general

Reply:
yes you are right from now on I use array functions tnx